editor/osm_editor.cpp

```diff
@@ -168,7 +170,11 @@ void Editor::LoadEdits()
   m_features.Set(make_shared<FeaturesContainer>());
   auto loadedFeatures = make_shared<FeaturesContainer>();
 
-  for (auto const & mwm : doc.child(kXmlRootNode).children(kXmlMwmNode))
+  auto rootNode = doc.child(kXmlNewRootNode);
+  // Migrate clients with an old root node.
+  if (!rootNode)
+    rootNode = doc.child(kXmlOldRootNode);
```

Comment:
- I don't mind. Delete kXmlOldRootNode var and put _old_ string manually here?
- add CHECK(rootNode, ());